doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1466,12 +1480,46 @@ Response:
 
 + Response 204
 
-## OMA-NGSI Operations [/v2/queryContext]
+## POJ RPC Operations [/v2/op]
 
-For the sake of completeness here is an enumeration of the OMA-NGSI (9 & 10) operations. These
-operations will be supported under the 'v2' resource as well.
+### Update [POST /v2/op/update]
+
+This operation allows to create, update and/or delete several entities in a single batch operation.
+The payload is an object with two properties:
+
+* `actionType`, to specify the kind of update action to do: either APPEND, APPENDONLY, UPDATE, DELETE.
+* `entities`, an array of entities, each one specified using the JSON entity representation format (described
+  in a section above).
+
+Response:
+
+* Successful operation uses 204 No Content.
+* Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
+
```

Comment:
I think here notifying errors can be a bit more complex ... for instance what happens if only one of the affected entities was updated and the rest not ... Are these operations expected to work as if they were transactions?